platform/android/src/native_map_view.cpp

```diff
@@ -73,6 +55,9 @@ NativeMapView::NativeMapView(jni::JNIEnv& _env,
         pixelRatio, mbgl::android::FileSource::getDefaultFileSource(_env, jFileSource), *threadPool,
         MapMode::Continuous, GLContextMode::Unique, ConstrainMode::HeightOnly,
         ViewportMode::Default, jni::Make<std::string>(_env, _programCacheDir));
+
+    _initializeDisplay();
+    _initializeContext();
```

Comment:
current codebase always requires to have one map object for one display/context. This is now consolidated in the constructor of the map object. This as a result removes `NativeMapView::initializeDisplay`, ` NativeMapView::terminateDisplay`, ....